File: remote/src/main/java/com/kelsos/mbrc/models/NotificationModel.java
package com.kelsos.mbrc.models;

import android.graphics.Bitmap;
import com.kelsos.mbrc.annotations.PlayerState;
import com.kelsos.mbrc.dto.track.TrackInfo;

public class NotificationModel {
  private TrackInfo trackInfo;
  private Bitmap cover;
  @PlayerState.State private String playState;

  public NotificationModel() {
    playState = PlayerState.STOPPED;
  }

  public TrackInfo getTrackInfo() {
    return trackInfo;
  }

  public void setTrackInfo(TrackInfo trackInfo) {
    this.trackInfo = trackInfo;
  }

  public Bitmap getCover() {
    return cover;
  }

  public void setCover(Bitmap cover) {
    this.cover = cover;
  }

  @PlayerState.State public String getPlayState() {
    return playState;
  }

  public void setPlayState(@PlayerState.State String playState) {
    this.playState = playState;
  }
}

File: remote/src/main/kotlin/com/kelsos/mbrc/models/NotificationModel.kt
package com.kelsos.mbrc.models

import android.graphics.Bitmap
import com.kelsos.mbrc.annotations.PlayerState
import com.kelsos.mbrc.domain.TrackInfo

class NotificationModel {
  var trackInfo: TrackInfo = TrackInfo()
  var cover: Bitmap? = null
  @PlayerState.State var playState: String

  init {
    playState = PlayerState.STOPPED
  }
}
